File: 12.2) 1030A - 705A - 486A - 61A - 200B/Özgün/InSearchOfAnEasyProblem.cpp
#include <iostream>

using namespace std;

int main() {
  int n, d;
  bool isEasy = true;

  cin >> n;

  while (n--) {
    cin >> d;

    if (d == 1) {
      cout << "HARD";
      isEasy = false;
      break;
    }
  }

  if (isEasy) cout << "EASY";

  return 0;
}

File: 12.2) 1030A - 705A - 486A - 61A - 200B/Mehmet Berat Ersarı/InSearchofanEasyProblem.java
import java.util.Scanner;

public class InSearchofanEasyProblem {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n =sc.nextInt();
        int easy=0;
        int hard=0;
        for(int i=0;i<n;i++){
            if(sc.nextInt()==0)
                easy++;
            else
                hard++;
        }
        if(hard>0)
            System.out.println("HARD");
        else
            System.out.println("EASY");



    }
}
